circle.yml

```diff
@@ -2671,22 +2671,19 @@ windows-workflow: &windows-workflow
           - windows-build
 
     - lint:
-        <<: *full-windows-workflow-filters
         name: windows-lint
         executor: windows
         requires:
           - windows-build
 
     - unit-tests:
-        <<: *full-windows-workflow-filters
         name: windows-unit-tests
         executor: windows
         resource_class: windows.large
```

Comment:
Do we want to back this off to medium?  Or do we want to explore that separately?

Reply:
Right now it looks like it's still too flaky :confused: https://github.com/cypress-io/cypress/pull/22821

Reply:
Sadness. Oh well this is a good option for now! 👍🏻